### PR TITLE
Bandanas/More jumpsuits in mixed fixed

### PR DESCRIPTION
**Pull request by dannno**
Mainly bandanas

just some bandanas that spawn in various lockers for fun. 
I also removed an extraneous lightblue jumpsuit from the code and added more jumpsuit colors to the mixed lockers.

I plan to add more kinds of bandanas in the future when I get the sprites.

sorry for all the commit nonsense :]

### CODE REVIEW

**Comment by dannno** [comment]
this is all good to be merged and has technically already been around for more than 24h

**Comment by dannno** [comment]
for the love of god, montresor

**Comment by Jordie0608** [comment]
Changelog

**Comment by dannno** [comment]
what

>just some bandanas that spawn in various lockers for fun. 
I also removed an extraneous lightblue jumpsuit from the code and added more jumpsuit colors to the mixed lockers.

**Comment by Jordie0608** [comment]
A yml you dork.

**Comment by dannno** [comment]
can you just speak in real words maybe

**Comment by Boggart** [comment]
[Here's what he's talking about.](https://tgstation13.org/wiki/Guide_to_Changelogs)

**Comment by dannno** [comment]
it's good 2 go now i'm 99% sure

**Comment by RemieRichards** [comment]
@dannno Conflicts bby.

**Comment by Boggart** [comment]
I'm not sure if you're supposed to change the PR part of the changelog name or not. The ones I looked at as an example when I did mine used <username>-PR-<ID>.yml

**Comment by dannno** [comment]
nigga what
what conflicts

**Comment by paprka** [comment]
Please test and make sure the mixed lockers can contain all of this shit without being overstuffed

**Comment by dannno** [comment]
they can. I do test.

File: config/admins.txt [hunk 1]
```diff
@@ -73,6 +73,10 @@ Cuboos = Game Master
 thunder12345 = Game Master
 wjohnston = Game Master
 mandurrh = Game Master
+<<<<<<< HEAD
+dannno = Game Master
+=======
 thurgatar = Game Master
 xerux = Game Master
 dannno = Game Master
+>>>>>>> d5950a62a4f4208a1be32a3424765f23174b9db7
```
Comment (tkdrg) [inline config/admins.txt]: Conflicts weren't fixed properly here. Remove the git marker tags and adjust the contents as needed.

**Comment by tkdrg** [comment]
Did you look into doing the thing I mentioned in the previous PR about multi-slot equipping?

**Comment by TZK13** [comment]
I can always do that after this is merged if need be considering I'm the one that sort of brought it up.

File: config/admins.txt (hunk 1 above)
Comment (dannno) [inline config/admins.txt]: HRFF

**Comment by dannno** [comment]
@tkdrg i totally forgot like a dumbo idiot and will do that because it's a gr8 idea

**Comment by dannno** [comment]
ok this is actually for real really done this time now for rizzle
:)

File: code/modules/clothing/clothing.dm [hunk 2]
```diff
@@ -85,6 +85,7 @@ BLIND     // can't see anything
 	put_on_delay = 40
 	var/mask_adjusted = 0
 	var/ignore_maskadjust = 1
+	var/hathead_only = 1
```
Comment (tkdrg) [inline code/modules/clothing/clothing.dm]: This is snowflake. It only works for masks that can also be hats. I suggest a more general system. Something like `var/adjusted_flags = null` and then on the adjust_mask code:
```
if(adjusted_flags)
    slot_flags = adjusted_flags
```

File: code/modules/clothing/clothing.dm [hunk 3]
```diff
@@ -111,6 +113,8 @@ BLIND     // can't see anything
 			flags &= ~visor_flags
 			flags_inv &= ~visor_flags_inv
 			src.mask_adjusted = 1
+			if(!hathead_only)
+				slot_flags = SLOT_HEAD | SLOT_MASK
```
Comment (tkdrg) [inline code/modules/clothing/clothing.dm]: This means the head-only bandana can be worn on the mask slot, which is really weird and makes no sense.

Comment (dannno) [inline code/modules/clothing/clothing.dm]: I MAINTAIN THAT IT IS SO THAT YOU CAN WEAR A DURAG UNDER YOUR HAT

**Comment by dannno** [comment]
this time im really not lying for real

**Pull request merged**